.cg-docs/work-reports/2026-08-07-copilot-pilot-evidence.md

```
@@ -0,0 +1,366 @@
+- Full `statusCheckRollup` captured via GraphQL during closeout. Summary:
+
+| Check | Status | First seen | Conclusion |
+|-------|--------|-----------|------------|
+| Browser evidence manifest tests | COMPLETED | 12:40:11Z | SUCCESS |
+| Docs staleness check | COMPLETED | 12:40:12Z | SUCCESS |
+| Link Check | COMPLETED | 12:40:10Z | SUCCESS |
+| **Native target Python gate on macos-14** | COMPLETED | 12:40:11Z | SUCCESS |
+| **Native target Python gate on windows-2022** | COMPLETED | 12:40:12Z | SUCCESS |
+| **Pester on macos-14** | COMPLETED | 12:40:17Z | SUCCESS |
+| **Pester on windows-2022** | COMPLETED | 12:40:11Z | SUCCESS |
+| **PR title follows Conventional Commits** | COMPLETED | 12:40 (fail x3) → 13:08/13:09 (pass) | SUCCESS (final) |
```

markdownlint-cli2 (0.23.2)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)